django_cas_ng/backends.py

```diff
@@ -44,6 +44,9 @@ def authenticate(self, ticket, service, request):
             user.save()
             created = True
 
+        if not self.check_additional_permissions(user):
+            return False
```

Comment:
**authenticate()** need return an object.
You can return None or throw some kind of exception.